app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="form-field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="form-field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="form-field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] The views in devise for the registration and login should have been modified to match the wireframes and their styling.

### Your design 

### Sign in page 
![Screenshot from 2022-03-02 03-04-04](https://user-images.githubusercontent.com/22434660/156253254-1d6608bd-e22d-4f82-a127-e283b2b57770.png)


### Sign-Up Page

![Screenshot from 2022-03-02 03-04-15](https://user-images.githubusercontent.com/22434660/156253304-dd43dbf5-0bf4-496c-b7b5-0259e9ab2d06.png)


### Wireframe design

![Screenshot from 2022-03-02 03-09-41](https://user-images.githubusercontent.com/22434660/156253415-3f9ec770-583b-46df-81ef-2dca6284a55f.png)

Reply:
Thank you @Dipeshtwis. I will work on this 🙏